gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -291,7 +291,7 @@ export default function NxGlobalSidebarPage() {
         <NxTile.Content>
           <NxP>
             This examples shows 10 nav items (the maximum), a middle content area with enough content to trigger
-            scrolling, and all footer options enabled.
+            scrolling, all footer options enabled.
```

Comment:
Why'd you get rid of "and" here?

Reply:
> Why'd you get rid of "and" here?

Definitely an accident when I was thinking about whether I should add a comment in about the logo size here. [d737a1d](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/d737a1dd6adfe253aff5e9435779531dddff5ac9)